reporting/spacewalk-report

```diff
@@ -301,17 +301,17 @@ if __name__ == '__main__':
                         synopsis = report.synopsis
                     except:
                         None
-                    print "%s: %s" % (report_name, synopsis)
+                    print("%s: %s" % (report_name, synopsis))
```

Comment:
If we do that modern way, then let's use `format` instead:
```python
print('{}: {}".format(report_name, synopsis))
```

Reply:
No general refactoring